web/modules/custom/dpl_example_content/content/eventseries/2ce9d9b0-7f10-4a17-9a5e-383d36c1ed93.yml

```diff
@@ -118,6 +118,11 @@ default:
   field_event_image:
     -
       entity: a0176f8c-bb7e-4f5a-bf23-b48cd8bab05f
+  field_event_link:
+    -
+      uri: 'https://www.google.com/'
```

Comment:
Consider using a more appropriate link like https://place2book.com/ or https://billetto.dk/ for these links.

I know we have been tossing around google.com in other cases but now that we store in content that is reused I think it makes sense to use values that make a little more sense.